src/profile/style.scss

```diff
@@ -0,0 +1,39 @@
+.gp-settings {
+	&__credits {
+		a {
+			text-decoration: none;
+
+			&:hover {
+				text-decoration: underline;
+			}
+		}
+	}
+
+	&__credits-featured {
+		display: flex;
+		flex-wrap: wrap;
+		gap: 1rem;
+		margin-bottom: 2rem;
+
+		a {
+			display: block;
+			font-size: 1.2rem;
+			font-weight: 600;
+			padding: 1rem;
+		}
+
+		img {
+			border-radius: 999rem;
+			margin-bottom: 0.5rem;
+		}
+
+		li {
+			text-align: center;
+			width: 10rem;
+		}
+	}
```

Comment:
This doesn't look right. Credits should be a setting thing, but not needed here.

Reply:
Yeah, I don't think any styles are needed. Im going to leave an empty file for now.